src/rules/at-rule-empty-line-before/index.js

```diff
@@ -74,5 +75,17 @@ export default function (expectation, options) {
         )
       }
     })
+
+    validateOptions({ result, ruleName,
```

Comment:
That's fine by me. I tried both places. I would really prefer to put it before immediately inside the outer function (before `return (root, result) => { .. }`); but we need a result to attach the warnings to. I'll make this adjustment.